src/components/BookForm.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export default function BookForm() {
+  return (
+    <form>
+      <input type={Text} placeholder="Add a title" />
+      <input type={Text} placeholder="Add an Author" />
+      <button type="submit">Submit</button>
+    </form>
+  );
+}
```

Comment:
- [ ] Create Job Creating BookForm component 👏 , but as part of the requirement it should be called inside the Books components which is missing, anyway you should follow this flow with components : 

```JS
//Books.js the main component that should be called in the '/' root
export default function Books() {
  const books = [] //add some books in the array
  return (
<>
    {books.map(data =>(
    <Book key={data.id} data={data} />
    )}
   <BookForm /> //the form you already created should be called here
</>
);
}
```

```JS
//Book.js The component that takes book data as props and displays a book
export default function Book({data}) {
  return (
<>
 // show books data here
// add delete book button as well (doesn't have to work its just mockup for now)
</>
);
}
```

Please update your code to match the requirements 🙏 

**_🛑 Note: you don't have to do exactly what i typed , its just an example to help you understand the requirements feel free to produce your own code 🙏_**